pkg/util/ip/ip.go

```diff
@@ -141,8 +151,19 @@ func IPNetToNetIPNet(ipNet *v1beta1.IPNet) *net.IPNet {
 	return &net.IPNet{IP: ip, Mask: net.CIDRMask(int(ipNet.PrefixLength), bits)}
 }
 
-// Function to transform net.IPNet to Antrea IPNet
+// NetIPNetToIPNet transforms net.IPNet to Antrea IPNet
 func NetIPNetToIPNet(ipNet *net.IPNet) *v1beta1.IPNet {
 	prefix, _ := ipNet.Mask.Size()
 	return &v1beta1.IPNet{IP: v1beta1.IPAddress(ipNet.IP), PrefixLength: int32(prefix)}
 }
+
+// LookupProtocolMap return protocol identifier given protocol name
+func LookupProtocolMap(name string) (uint8, error) {
```

Comment:
If you are using this function in a single place, I would suggest keeping it in the same file as the caller for now